recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set gfortran_version = [7, 5, 0] %}
+{% set gfortran_version = [9, 3, 0] %}
 {% set libgfortran_version = [4, 0, 0] %}
 {% set max_libgfortran_version = "5.0.0.a0" %}
```

Comment:
You're going to need get the SO version for libgfortran and put it here.

Reply:
I am still a bit confused can you point me to the corresponding Linux recipe?

Reply:
If you look at the tests below, there is a SO version in the file path of the output libraries. We need to put this major version here. I presume if we bump by one (so 5 to 6) it will match. 

The linux builds are totally different so those are not a good reference.

Reply:
It seems to be 5 - at least in the `gcc_impl_linux` package `9.3.0` there is a `libgfortran.so.5.0.0`

Reply:
yup so change the version to 5 and the max version to 6

Reply:
ahhhh missed the commit! Cheers!